src/popup/darkMode.tsx

```diff
@@ -0,0 +1,35 @@
+export function darkMode(popupWindow: Window) {
```

Comment:
Not a big fan of mixing CSS and JavaScript. What do you think about changing `colors.tsx` to be:
```
$human-blue: #164176;
$human-pink: #fa759e;
$human-yellow-warning: #7e5e00;

.light-theme {
  --background-color: white !important;
  --text-color: #{$human-blue};
  --post-btn-background-color: #{$human-blue};
  --post-btn-text-color: white;
  --image-border-color: #{$human-blue};
  --emoji-picker-border-color: #d9d9d9;
}

.dark-theme {
  --background-color: #{$human-blue} !important;
  --text-color: white;
  --post-btn-background-color: #{$human-pink};
  --post-btn-text-color: #{$human-blue};
  --image-border-color: #{$human-pink};
  --emoji-picker-border-color: 'transparent';
}
```

Updating the `App` component to:
```
export function App(props: AppProps): JSX.Element {
  const theme = props.storeState.darkModeOn ? 'dark-theme' : 'light-theme'
  return (
    <div className={`app ${theme}`} >
      <AppContents {...props} />
    </div >
  )
}
```

And adding the background to `.app` instead of `root, body`:
```
.app {
  background-color: var(--background-color);
}
```

Reply:
Of course, we can use Sass variables instead of CSS variables

Reply:
Love this approach! Just pushed those changes. I added a blank `<div class="app"></div>` to the popup.html so that the app would be present and have the proper color while the app is loading.

Reply:
Ended up doing something extremely similar, but used an implicit light theme so that the blank `<div class="app"></div>` in the popup.html gets the default theme